addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1544,9 +1600,15 @@ var SnippetsMenu = Widget.extend({
                 }
 
                 self._activateSnippet(false);
+                self._insertDropzoneForbidSanitize = $snippet.data('oeForbidSanitize');
                 self._activateInsertionZones($selectorSiblings, $selectorChildren);
+                delete self._insertDropzoneForbidSanitize;
```

Comment:
Looks quite unusual. That's something done that way but to get rid in master? Write a comment please
// Done this way in stable, will be refactored in master